.storybook/index.js

```diff
@@ -59,14 +59,14 @@ const disableCentered = stories =>
 /**
  * Configuration for applying information to individual stories.
  * @param {string} description The information to apply to individual stories.
- * @param {Object.<string, string>} carbon - An object containing Carbon information to redirect to.
+ * @param {{library: string, story: string, id: string}} component An object containing component information to redirect to.
  * @returns {Object<string, string>} The configuration containing information to apply.
  */
-const info = (description, carbon = null) => ({
-  info: carbon
+const info = (description, component = { library: 'react' }) => ({
+  info: component.story
     ? `${description}
 
-Also refer to http://react.carbondesignsystem.com/?path=/story/${carbon.story}--${carbon.id}`
+Also refer to http://${component.library}.carbondesignsystem.com/?path=/story/${component.story}--${component.id}`
```

Comment:
I'm good with this change, but it's no longer working as expected in existing implementations of `Accordion` and `UIShell`:

<img width="1680" alt="Accordion" src="https://user-images.githubusercontent.com/3846874/80798335-16afcb00-8b9c-11ea-9238-e0c1866447e2.png">